traveler-swift-core/TravelerKit/Delegates/CancellationDelegate.swift

```diff
@@ -12,8 +12,10 @@ import Foundation
 public protocol CancellationDelegate: class {
     /**
      Called when the `Order` was cancelled successfully
+      - Parameters:
+     -order: The `Order`that was cancelled
      */
-    func cancellationDidSucceed()
+    func cancellationDidSucceed(_ order: Order)
```

Comment:
`func cancellationDidSucceed(order: Order)`

Only omit variable names if its clear in the signature what the variable is or if the language reads fluently